.buildkite/build-static-release.sh

```diff
@@ -74,14 +75,17 @@ else
 fi
 rbenv exec gem install sorbet-*.gem
 
-mkdir -p srb-init-smoke-test
-(
-  cd srb-init-smoke-test
-  SRB_YES=1 rbenv exec srb init
-  rbenv exec srb tc -e 'puts 1'
+smoke_test_dir="$(mktemp -d)"
+# Explicitly want this string to expand eagerly.
+# shellcheck disable=SC2064
+trap "rm -rf '$smoke_test_dir'" EXIT
 
-  rm -rf sorbet/
+cp ../../.ruby-version "$smoke_test_dir"
 
+(
+  # Make sure we're in a totally separate tree, otherwise a Gemfile from a
+  # parent folder will affect this test.
+  cd "$smoke_test_dir"
   touch Gemfile
```

Comment:
Do we want to support using sorbet outside of a bundler project?

Reply:
Every phase of `srb` needs to know what gems you're using. If you don't have a Gemfile that declares them, we'd have to like search your gem path to find what you *could* potentially require. I think that `srb tc` should still work regardless.

But at this point i think bundler is so ubiquitous that it's not a high priority.

Reply:
That should also be separate from improving this test. I'm happy to have an issue to track it, but i would kind of prefer to have someone ask for it first so we can really understand what they're asking for.

Reply:
That makes sense, thanks!